Libs/MRML/Widgets/qMRMLVolumeThresholdWidget.cxx

```diff
@@ -60,10 +60,6 @@ void qMRMLVolumeThresholdWidgetPrivate::init()
 
   this->Superclass::init();
   this->setupUi(q);
-  this->VolumeThresholdRangeWidget->minimumSpinBox()->setDecimalsOption(
-    ctkDoubleSpinBox::DecimalsByKey|ctkDoubleSpinBox::DecimalsByShortcuts);
-  this->VolumeThresholdRangeWidget->maximumSpinBox()->setDecimalsOption(
-    ctkDoubleSpinBox::DecimalsByKey|ctkDoubleSpinBox::DecimalsByShortcuts);
```

Comment:
Why was this deleted? Is it now set in the .ui file?

Reply:
I believe this was an accident when I first started working on this. I have restored this code.